presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestIntegrationSmokeTest.java

```diff
@@ -28,7 +28,7 @@
 {
     protected AbstractTestIntegrationSmokeTest(QueryRunnerSupplier supplier)
     {
-        super(supplier);
+        super(supplier, FeatureSelection.all());
```

Comment:
nit:
`AbstractTestIntegrationSmokeTest` doesn't make use of  feature selection (if it did, a constructor accepting feature selection would be needed). Therefore the superclass (`AbstractTestQueryFramework`) should still provide a constructor that doesn't require passing feature selection.